getting-started.ipynb

```diff
@@ -0,0 +1,528 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Getting Started"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Welcome to **PyTorch-Ignite**’s quick start guide that covers the\n",
+    "essentials of getting a project up and running while walking through\n",
+    "basic concepts of Ignite. In just a few lines of code, you can get your\n",
+    "model trained and validated. The complete code can be found at the end\n",
+    "of this guide."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "This tutorial assumes you are familiar with the:\n",
+    "\n",
+    "1.  Basics of Python and deep learning\n",
+    "2.  Structure of PyTorch code"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Installation\n",
+    "\n",
+    "1.  From `pip`\n",
+    "\n",
+    "``` shell\n",
+    "pip install pytorch-ignite\n",
+    "```\n",
+    "\n",
+    "1.  From `conda`\n",
+    "\n",
+    "``` shell\n",
+    "conda install ignite -c pytorch\n",
+    "```\n",
+    "\n",
+    "See [here](/docs/how-to-guides/installation) for other installation\n",
+    "options."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Code\n",
+    "\n",
+    "Import the following:"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "source": [
+    "import torch\n",
+    "import torch.nn.functional as F\n",
```

Comment:
I was also wanted to get rid of the calls of functional part and use `nn.*` instead. Let's do that here.

Reply:
Removed this.